### DISCUSSION TITLE
Installer

### DISCUSSION THREAD

**Discussion by ghost** [atom/atom#23587] [General]
I tried to install atom on Windows 7.
It gave me an error like "Contact with application`s authors".
Can you suggest anything?

Looking through the application code,
I found that the authors do not generate installation files using the Electron library.
Or I'm wrong?

I ran into the same problem when
I was developing my HummyIDE project.

And what i must do? I dont know :(
Can Atom developers generate a installation files with an electron-builder`s library?
Or with some other?

**Comment by ghost** [comment]
Note: I tried to using .NET 4.5.2 and 4.5.3
But nothing :(

**Comment by icecream17** [comment]
{ignore, I don't know how to debug this, I'm just linking to an issue https://github.com/atom/atom/issues/21769}
